### ISSUE TITLE
2 questions

### ISSUE THREAD

**Issue by janwillemtulp** [sveltejs/svelte-scroller#4]
1. what's going on with these lower case element names, like `<svelte-scroller-outer>` that only seem to map to css classes?

2. what is the best way to allow pointer events for the fixed part without changing the original code? Currently pointer-events are disabled, but I have an interactive component in my fixed part.

**Comment by basilesimon** [comment]
> what is the best way to allow pointer events for the fixed part without changing the original code? Currently pointer-events are disabled, but I have an interactive component in my fixed part.

I'm currently wondering the same thing :)

**Comment by njdehoog** [comment]
> what is the best way to allow pointer events for the fixed part without changing the original code? Currently pointer-events are disabled, but I have an interactive component in my fixed part.

@janwillemtulp were you ever able to figure this out? I'm running into the same issue.

**Comment by jeffmacinnes** [comment]
Has anyone had any luck figuring out a workaround for this? I'm hitting the same issue as well....

**Comment by njdehoog** [comment]
Yes, here's a demo project: https://svelte.dev/repl/32abb5e2ab3a4670a19206a40ab7f375?version=3.55.1

You need the following CSS:
```
[slot="background"] {
  pointer-events: all;
}

[slot="foreground"] {
  pointer-events: none;
}
```

And then you need to make sure the parent doesn't 'eat' the pointer events:
```
.your-container-div {
  pointer-events: none;
}
```

**Comment by jeffmacinnes** [comment]
@njdehoog Amazing! Thank you so much!
